android/app/src/main/java/org/fossasia/openevent/core/track/session/SessionDetailActivity.java

```diff
@@ -413,7 +414,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 intent.putExtra(CalendarContract.Events.DESCRIPTION, Html.fromHtml(session.getShortAbstract()).toString());
                 intent.putExtra(CalendarContract.EXTRA_EVENT_BEGIN_TIME, DateConverter.formatDateWithDefault(DateConverter.FORMAT_24H, session.getStartsAt()));
                 intent.putExtra(CalendarContract.EXTRA_EVENT_END_TIME, DateConverter.formatDateWithDefault(DateConverter.FORMAT_24H, session.getEndsAt()));
-                startActivity(intent);
+                if (intent.resolveActivity(getPackageManager()) != null){
```

Comment:
Space after `)`

Reply:
which ) do you want the change? The ones in the if statement?